### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by mariaalouisaa**
- I linked fontAwesome to the index.html.
- Changed the button plus text to a plus icon.
- Gave it an aria-label and tested with the screen reader - works well.
- Tweaked the height of the input + button so they are the same.
- Remove the old font from the button (as it's not needed there anymore).

Can use this same method for the buttons later too!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -55,7 +55,8 @@ function createList() {
     //create button to submit input.
     const inputBtn = document.createElement('button')
     inputBtn.id="add-title"
-    inputBtn.innerHTML="Add"
+    inputBtn.innerHTML=`<i class="fas fa-plus"></i>`
```
Comment (PJSalter) [inline index.js:58]: 🎊 🥳

File: index.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/22ca1cdaec.js" crossorigin="anonymous"></script>
```
Comment (PJSalter) [inline index.html:7]: Thank You for adding the buttons from fontawesome 🤩

File: style.css [hunk 3]
```diff
@@ -197,12 +197,11 @@ h1 {
   cursor: pointer;
   outline: none;
   width: 15%;
-  height: 36px;
+  height: 39px;
```
Comment (PJSalter) [inline style.css:200]: a clear adjustment, great work ✨

**Review by PJSalter — APPROVED** [review]

File: style.css [hunk 4]
```diff
@@ -168,7 +168,7 @@ h1 {
 
 .list-input input{
  width: 75%;
- height: 30px;
+ height: 15px;
```
Comment (AsmahanM) [inline style.css:171]: Thanks for making these adjustments!

**Review by AsmahanM — APPROVED** [review]

**Pull request merged**